products/pan-os.md

```diff
@@ -36,8 +36,8 @@ releases:
 -   releaseCycle: "11.1"
     releaseDate: 2023-11-03
     eol: 2026-11-03
-    latest: "11.1.3-h6"
-    latestReleaseDate: 2024-09-05
+    latest: "11.1.4-h1"
+    latestReleaseDate: 2024-08-08
```

Comment:
the date got changed @mrjcap 

also looks like the automation cannot handle the link well.

Reply:
@chenrui333, Yes. That is correct.